### PR TITLE
Snapshot update

### PR DESCRIPTION
**Pull request by PavelBal**
Add installation of package `showtext`

### CODE REVIEW

**Comment by Felixmil** [comment]
🥳

File: DESCRIPTION [hunk 1]
```diff
@@ -35,7 +35,9 @@ Suggests:
     rmarkdown,
     testthat (>= 3.0.3),
     vdiffr (>= 1.0.0),
-    withr
+    withr,
+    showtext,
```
Comment (Felixmil) [inline DESCRIPTION:39]: Suggest are not supposed to propagate to depending package but well... I guess this is a special case.

**Review by Felixmil — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (PavelBal) [reply DESCRIPTION:39]: > Suggest are not supposed to propagate to depending package but well... I guess this is a special case.

I was desperate... but it did actually help. Could you please add it to suggest of esqlabsR?

**Comment by PavelBal** [comment]
@Felixmil could you approve once more, please? I removed the explicit installation of the aforementioned packages from the AppVeyor script (works just fine)

**Review by Felixmil — APPROVED** [review]

**Pull request merged**